src/tracer.js

```diff
@@ -54,11 +55,13 @@ export default class Tracer {
      *        as process-level tags on the Tracer itself.
      * @param {Object} [options.metrics] - instance of the Metrics class from ./metrics/metrics.js.
      * @param {Object} [options.logger] - a logger matching NullLogger API from ./logger.js.
+     * @param {Boolean} [upsampling] - determines whether to resample child spans of unsampled parents.
      */
     constructor(serviceName: string,
             reporter: Reporter = new NoopReporter(),
             sampler: Sampler = new ConstSampler(false),
-            options: any = {}) {
+            options: any = {},
+            upsampling: any = {}) {
```

Comment:
upsampling should be passed as part of the `options`: `options.upsampling.enabled`